app/Http/helpers.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+use Illuminate\Http;
+
+function set_active_link($path, $active = '')
+{
+    return Request::is($path) ? $active : '';
+}
```

Comment:
Yeah mainly for view only. Got this idea from a laracasts video (may not be the most up-to-date way of doing this though...)